browsers/edge.json

```diff
@@ -360,13 +360,15 @@
         },
         "123": {
           "release_date": "2024-03-22",
-          "status": "current",
+          "release_notes": "https://learn.microsoft.com/en-us/deployedge/microsoft-edge-relnote-stable-channel#version-1230242081-april-4-2024",
```

Comment:
https://learn.microsoft.com/en-us/deployedge/microsoft-edge-relnote-stable-channel#version-1230242053--march-22-2024 is the oldest for 123 and matches the date above.